refinery/ui/source/js/data-set-import/partials/metadata-table.html

```diff
@@ -31,14 +31,14 @@
             Can have additional columns used to characterize data
           </li>
           <li>
-            Can have the columns ordered arbitrary
+            Can have the columns ordered arbitrarily
```

Comment:
According to our copy style guide we should use active voice: "Can have arbitrary column order".